decidim-accountability/spec/shared/manage_child_results_examples.rb

```diff
@@ -39,7 +39,7 @@
   end
 
   it "creates a new child result" do
-    click_link "New Result"
+    click_link "New Result", match: :first
```

Comment:
Why this?

Reply:
Since we're now matching links by title instead of raw markup, in this case we now have two links called "New result". The "call to action" button to the right of the title and one of the actions with a plus icon. I think they represent creating results at different levels, not sure if this represents a real UX problem and it should be clarified somehow.